src/lsp/cobol_ptree/statements_visitor.ml

```diff
@@ -45,6 +45,7 @@ class virtual ['a] folder = object
   method fold_display_clause'     : (display_clause with_loc , 'a) fold = default
   method fold_display_target'     : (display_target with_loc , 'a) fold = default
   method fold_display_with_clause': (display_with_clause with_loc, 'a) fold = default
+  method fold_entry_by_clause     : (entry_clause         , 'a) fold = default
```

Comment:
Shouldn't this be `(entry_by_clause, 'a) fold`? Otherwise the method is misnamed.